src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -1269,6 +1283,15 @@ GlobeControls.prototype.getCameraTargetPosition = function getCameraTargetPositi
 GlobeControls.prototype.setCameraTargetPosition = function setCameraTargetPosition(position, isAnimated) {
     const center = this.getCameraTargetPosition();
 
+    if (position.range) {
+        // Compensation of the altitude from the approximation of the ellipsoid by a sphere
```

Comment:
Then the question is: where do we approximate the ellipsoid by a sphere that in turn make this hack necessary?